concent_api/core/message_handlers.py

```diff
@@ -695,6 +699,12 @@ def handle_send_force_subtask_results_response(
             deserialize_message(subtask.task_to_compute.data.tobytes()),
         ])
 
+        delete_deposit_claim(
+            subtask_id=task_to_compute.subtask_id,
+            concent_use_case=ConcentUseCase.FORCED_ACCEPTANCE,
```

Comment:
Please use `discard_claim()`. `delete_deposit_claim()` does not lock the `DepositAccount`.

Reply:
:+1: done